spec/models/comfy/cms/file_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Comfy::Cms::File do
+  describe '#styles' do
+    context 'various dimensions' do
+      {
+        extra_small: '390x244',
+              small: '485x304',
+             medium: '900x564',
+              large: '1350x846'
```

Comment:
This line does not have a proper indentation with the first item on the hash. Can you run the rubocop on the files changed on the PR, please?